backend/routers/backups.py

```diff
@@ -39,11 +40,11 @@ def backup(data: dict, uid: str):  # Depends(auth.get_current_user_uid)
 
 
 @router.get('/v1/backups', tags=['v1'])
-def get_backup(uid: str):  # Depends(auth.get_current_user_uid)
+def get_backup(uid: str = Depends(auth.get_current_user_uid)):
```

Comment:
Don't modify backend existing endpoints, otherwise, will not work for current users, when we deploy the backend.

Reply:
Probably adding a v2 makes more sense

Reply:
backup endpoints are not needed anymore, so you can ignore this one.

Reply:
So should I revert these changes or just keep them as they are currently?

Reply:
For backups yes, you can remove the changes, same for plugins

Reply:
For the v1 routes of plugins, these changes were already there, so I'm not removing them or should I remove them?